dispatch/static/manager/src/js/components/Header.js

```diff
@@ -54,6 +54,9 @@ export default function Header(props) {
                 <li className='c-header__dropdown__link'>
                   <Link to='/events/'><span className='pt-icon-standard pt-icon-timeline-events' /> events</Link>
                 </li>
+                <li className='c-header__dropdown__link'>
+                  <Link to='/events/audit'><span className='pt-icon-standard pt-icon-endorsed' /> audit events</Link>
+                </li>
```

Comment:
Instead of adding a separate link in the navigation, I think it would be better to link to this page from the main events listing. Something like this:

![screen shot 2017-06-13 at 9 16 06 am](https://user-images.githubusercontent.com/2547035/27084263-388cb77c-5019-11e7-90ec-4ef6c1d4f6a8.png)

Clicking on "5 pending events" would take the user to the approval page.

I used the Blueprint tag component with the following classNames:

```html
<div class="pt-tag pt-large pt-intent-warning">5 pending events</div>
```